examples/apps/data-object-grid/package.json

```diff
@@ -54,12 +56,12 @@
 		"@fluidframework/runtime-definitions": "workspace:~",
 		"@fluidframework/runtime-utils": "workspace:~",
 		"@fluidframework/view-adapters": "workspace:~",
-		"office-ui-fabric-react": "7.199.6",
```

Comment:
FluentUI has been repackaged as '@fluentui/react-components'.  'office-ui-fabric-react' is deprecated and unmaintained since v7 and has been forcing workarounds with the current TS compiler.